src/pickers/ClaimOfficerPicker.js

```diff
@@ -21,47 +23,50 @@ const ClaimOfficerPicker = (props) => {
   const modulesManager = useModulesManager();
   const { formatMessage } = useTranslations("claim", modulesManager);
   const [variables, setVariables] = useState({});
+  const dispatch = useDispatch();
 
-  const { isLoading, data, error } = useGraphqlQuery(
-    `
-    query ClaimOfficerPicker ($search: String) {
-      claimOfficers(search: $search, first: 20) {
-        edges {
-          node {
-            id
-            uuid
-            code
-            lastName
-            otherNames
-            ${extraFragment ?? ""}
-          }
-        }
-      }
-    }
-  `,
-    variables,
-    { skip: true },
-  );
+  const isLoading = useSelector((state) => state.claim.claimOfficers.isFetching);
+  const isLoaded = useSelector((state) => state.claim.claimOfficers.isFetched);
+  const options = useSelector((state) => state.claim.claimOfficers.items);
+  const error = useSelector((state) => state.claim.claimOfficers.error);
+
+  useEffect(async () => {
+     await dispatch(
+      fetchClaimOfficers(modulesManager, extraFragment, variables, {
+        first: 20,
+      }),
+    );
+  }, []);
+
+  const getOptionLabel = (option) => {
```

Comment:
If I get this right - this function is responsible for what's displayed in the picker. 
What is behavior when list of claim administrators is empty or when no entry is selected?

Reply:
![image](https://user-images.githubusercontent.com/58431955/211543877-6bffdb36-f8a5-48a7-ad97-1034dd8f9b1d.png)